### DISCUSSION TITLE
iotdb里面的time可以给它取别名吗？

### DISCUSSION THREAD

**Discussion by lxb9764** [apache/iotdb#13121] [General]
![bf70c45b001dfe432171b15cfe4785ce](https://github.com/user-attachments/assets/5ef70321-01da-4b72-970d-1ffb0960b726)
insert into root.iot_log.d2.d3(timestamp, s1, s2) values(1723098324072,3,3)
![ebbda4b31a4f3c443a4e9a34bc468516](https://github.com/user-attachments/assets/9c35ffb4-677e-4bd1-883d-9b067c40585d)

**Comment by CritasWang** [comment]
这个再当前模型中暂不支持，另外欢迎进去社区群聊沟通
